apps/aecontract/test/aecontract_SUITE.erl

```diff
@@ -3409,23 +3426,23 @@ sophia_state_gas(_Cfg) ->
 %%% Store
 %%%===================================================================
 
+store_from_map(Map) ->
+    maps:fold(fun aect_contracts_store:put/3,
+              aect_contracts_store:new(), Map).
+
+set_ct_store(Map, Ct) ->
+    aect_contracts:set_state(store_from_map(Map), Ct).
+
+get_ct_store(Ct) ->
+    aect_contracts_store:subtree(<<>>, aect_contracts:state(Ct)).
+
 create_store(_Cfg) ->
     state(aect_test_utils:new_state()),
     Acc1  = ?call(new_account, 100),
     Ct1   = ?call(insert_contract, Acc1, <<"Code for C1">>),
     Ct1   = ?call(get_contract, Ct1),
-    #{}   = aect_contracts:state(Ct1),
-    ok.
-
-update_store(_Cfg) ->
```

Comment:
Why is this test removed?

Reply:
As far as I could tell it was the same as `read_store`, but with fewer checks.